codalab/lib/bundle_cli.py

```diff
@@ -2446,7 +2446,16 @@ def print_basic_info(self, client, info, raw):
         show_hidden = client.fetch('user')['is_root_user']
 
         for key, value in worksheet_util.get_formatted_metadata(cls, metadata, raw, show_hidden):
-            lines.append(self.key_value_str(key, value))
+            if key == 'store':
```

Comment:
We actually shouldn't show this under the 'store' key for metadata because the 'store' key has a different meaning (it's the original bundle store that the bundle was uploaded to -- see https://github.com/codalab/codalab-worksheets/blob/c82a01e14272db382c460e0197ff813e7264282f/codalab/bundles/uploaded_bundle.py#L32).

Instead, we should:
- hide the store key from displaying -- that already happens because it's set to `hidden=True` in the link above
- below the metadata keys, add a separate section where we say "Bundle stores: store1, store2"

Reply:
Thanks, I have updated the code based on your suggestion:)

Reply:
thanks -- can you add a screenshot? :)

Reply:
Updated